### PR TITLE
Add BaseObject to Licence and Support models

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

File: src/ralph/licences/models.py [hunk 1]
```diff
@@ -53,11 +48,9 @@ class Meta:
 
 
 class Licence(
+    BaseObject,
     Regionalizable,
     AdminAbsoluteUrlMixin,
-    PermByFieldMixin,
-    TimeStampMixin,
-    TaggableMixin,
     models.Model,
```
Comment (mkurek) [inline src/ralph/licences/models.py]: try:

```
class Licence(Regionalizable, AdminAbsoluteUrlMixin, BaseObject)
```

File: src/ralph/supports/models.py [hunk 2]
```diff
@@ -31,12 +25,10 @@ class SupportStatus(Choices):
...
 
 class Support(
+    BaseObject,
     AdminAbsoluteUrlMixin,
     Regionalizable,
-    PermByFieldMixin,
     NamedMixin.NonUnique,
-    TimeStampMixin,
-    TaggableMixin,
     models.Model,
```
Comment (mkurek) [inline src/ralph/supports/models.py]: ditto

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**